less/navigation.less

```diff
@@ -1 +1,23 @@
-// Navigation Styles here
\ No newline at end of file
+nav {
+    // border:1px solid yellow;
+    width: 800px;
+    display:flex;
+    justify-content: space-between;
+    align-items:center;
+
+    h1 {
+        justify-self:flex-start;
+    }
```

Comment:
take a closer look at the design file and notice the spacing between this h1 and the links, you might need to reconsider your html structure of the `nav` to accomplish this, or altering the stylings here